### ISSUE TITLE
export POSIXLY_CORRECT considered harmful

### ISSUE THREAD

**Issue by mislav** [nvie/gitflow#289]
In e1ec57d48ac5c234262d84469af56b00b38929ab this line was added:

```
export POSIXLY_CORRECT=1
```

And it probably fixed some issues in git-flow, but it causes breakages down the line. Specifically, when git-flow does `git checkout`, that can trigger post-checkout git hooks. Some people in Rubyland have hooks that integrate with some Ruby-specific tools, and have rbenv (written in bash) to manage them. Because rbenv uses a construct (process substitution) that's not posix-compatible, this environment variable causes it to fail.

This was discovered in sstephenson/rbenv#222

In summary, because there might be other shell scripts down the line, git-flow should not mess up the environment for them. I propose either removing the `export`, or using the setting:

```
set -o posix
```

/cc @Thiana

**Comment by tejanium** [comment]
This causing asdf to fail as well https://github.com/asdf-vm/asdf/issues/581

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
